front/src/modules/trainschedule/components/ManageTrainSchedule/Map.tsx

```diff
@@ -103,6 +103,7 @@ const Map = ({
 
   const [mapIsLoaded, setMapIsLoaded] = useState(false);
 
+  const [operationnalPointId, setOperationnalPointId] = useState<string | undefined>(undefined);
```

Comment:
- add hovered in the name
- operational has only 1 n 😉
```suggestion
  const [hoveredOperationalPointId, setHoveredOperationalPointId] = useState<string | undefined>(undefined);
```

Reply:
Done